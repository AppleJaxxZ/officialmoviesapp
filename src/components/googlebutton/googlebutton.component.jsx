import googleimg from "./google.png";
import "./googlebutton.styles.scss";

export const GoogleButton = ({ ...props }) => {
  return (
    <div className="googlebutton" {...props}>
      <img src={googleimg} alt="googleimg" />
      <div>Sign in with Google</div>
      <div></div>
    </div>
  );
};

export default GoogleButton;
