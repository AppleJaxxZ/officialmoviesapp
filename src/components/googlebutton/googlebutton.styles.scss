.googlebutton {
  padding: 3px;
  margin: 10px;
  background-color: white;
  border: 1px solid black;
  text-align: center;
  border-radius: 5px;
  height: 40px;
  //   width: 150px;
  display: flex;
  cursor: pointer;
  justify-content: space-between;
  align-items: center;
  width: 200px;
  @media screen and (max-width: 960px) {
    width: unset;
  }
  img {
    background-color: red;
    height: 30px;
    width: 30px;
    @media screen and (max-width: 960px) {
      height: 30px;
      width: 30px;
    }
  }
  div {
    color: rgb(94, 91, 91);
    font-size: 15px;
    @media screen and (max-width: 960px) {
      font-size: unset;
    }
  }
}
