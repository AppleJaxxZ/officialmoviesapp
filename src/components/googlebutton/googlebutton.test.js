import React from "react";
import { GoogleButton } from "./googlebutton.component";
import { shallow } from "enzyme";

describe("GoogleButton should be rendered", () => {
  let wrapper;
  beforeEach(() => {
    const mockProps = { test: "string test" };
    wrapper = shallow(<GoogleButton {...mockProps} />);
  });

  it("Should snapshot GoogleButton", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
//        I am Alex
