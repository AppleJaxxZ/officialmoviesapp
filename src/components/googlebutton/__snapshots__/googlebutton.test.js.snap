// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleButton should be rendered Should snapshot GoogleButton 1`] = `ShallowWrapper {}`;
